initfiles/componentfiles/configxml/setvars_linux.xsl

```diff
@@ -42,108 +35,66 @@
+                        <xsl:with-param name="default" select="'20100'"/>
+                     </xsl:call-template>
+export localthorportinc=<xsl:call-template name="setOrDefault">
+                            <xsl:with-param name="attribute" select="@localThorPortInc"/>
+                            <xsl:with-param name="default" select="'200'"/>
+                        </xsl:call-template>
+export slavespernode=<xsl:call-template name="setOrDefault">
+                        <xsl:with-param name="attribute" select="@slavesPerNode"/>
+                        <xsl:with-param name="default" select="'1'"/>
+                     </xsl:call-template>
 export DALISERVER=<xsl:call-template name="getDaliServers">
                     <xsl:with-param name="daliServer" select="@daliServers"/>
                 </xsl:call-template>
-</xsl:if>
-<xsl:if test="string(@localThor) != ''">
-export localthor=<xsl:value-of select="@localThor"/>
-</xsl:if>
-<xsl:if test="string(@processPerSlave) != ''">
-export processperslave=<xsl:value-of select="@processPerSlave"/>
-</xsl:if>
-<xsl:if test="string(Storage/@breakoutLimit) != ''">
-export breakoutlimit=<xsl:value-of select="Storage/@breakoutLimit"/>
-</xsl:if>
-<xsl:if test="string(Storage/@refreshRate) != ''">
-export refreshrate=<xsl:value-of select="Storage/@refreshRate"/>
-</xsl:if>
-<xsl:if test="string($clusterType) != ''">
-export clustertype=<xsl:value-of select="$clusterType"/>
-</xsl:if>
-export replicateOutputs=<xsl:choose>
-           <xsl:when test="@replicateOutputs='true'">1</xsl:when>
-           <xsl:when test="@replicateOutputs='1'">1</xsl:when>
-           <xsl:otherwise>0</xsl:otherwise>
-            </xsl:choose>
+export localthor=<xsl:call-template name="setOrDefault">
+                    <xsl:with-param name="attribute" select="@localThor"/>
+                    <xsl:with-param name="default" select="'false'"/>
+                </xsl:call-template>
+export processperslave=<xsl:call-template name="setOrDefault">
+                            <xsl:with-param name="attribute" select="@processPerSlave"/>
+                            <xsl:with-param name="default" select="'true'"/>
+                       </xsl:call-template>
+export breakoutlimit=<xsl:call-template name="setOrDefault">
+                        <xsl:with-param name="attribute" select="Storage/@breakoutLimit"/>
+                        <xsl:with-param name="default" select="'3600'"/>
+                     </xsl:call-template>
+export refreshrate=<xsl:call-template name="setOrDefault">
+                        <xsl:with-param name="attribute" select="Storage/@refreshRate"/>
+                        <xsl:with-param name="default" select="'3'"/>
+                     </xsl:call-template>
 export autoSwapNode=<xsl:choose>
            <xsl:when test="SwapNode/@AutoSwapNode='true'">1</xsl:when>
            <xsl:when test="SwapNode/@AutoSwapNode='1'">1</xsl:when>
```

Comment:
Not new, but don't see why this property in particular vs others is checking '1' ,